Sources/StreamChatUI/Composer/ComposerView.swift

```diff
@@ -136,21 +136,30 @@ open class _ComposerView<ExtraData: ExtraDataTypes>: _View, ThemeProvider {
         headerView.addSubview(dismissButton)
 
         centerContainer.axis = .horizontal
-        centerContainer.alignment = .fill
+        centerContainer.alignment = .bottom
         centerContainer.spacing = .auto
         centerContainer.addArrangedSubview(leadingContainer)
         centerContainer.addArrangedSubview(inputMessageView)
         centerContainer.addArrangedSubview(trailingContainer)
+        
+        // The inputMessageView default height is 38pt,
+        // all the buttons in side containers (leadingContainer and trailingContainer) are 20pt.
+        // We want the buttons centered in the default state. (38-20)/2 = 9pt margin
+        let sideContainersBottomMargin: CGFloat = 9
```

Comment:
Maybe we can make this propery open? 🤔 It will be easier to customize it, instead of overriding both directionLayoutMargins in both trailing and leading containers

Reply:
Good catch! Thanks